airbyte-integrations/connectors/source-mixpanel-singer/integration_tests/standard_source_test.py

```diff
@@ -0,0 +1,29 @@
+"""
+MIT License
+
+Copyright (c) 2020 Airbyte
+
+Permission is hereby granted, free of charge, to any person obtaining a copy
+of this software and associated documentation files (the "Software"), to deal
+in the Software without restriction, including without limitation the rights
+to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+copies of the Software, and to permit persons to whom the Software is
+furnished to do so, subject to the following conditions:
+
+The above copyright notice and this permission notice shall be included in all
+copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+SOFTWARE.
+"""
+
+from base_python_test import DefaultStandardSourceTest
+
+
+class SourceMixpanelSingerStandardTest(DefaultStandardSourceTest):
```

Comment:
The standard source test should be implemented. `source-shopify-singer` has a good example of pointing at the files specified in `Dockerfile.test` for the test.

Reply:
FYI `DefaultStandardSourceTest` is a default implementation of the standard source test

Reply:
Sorry, Shrif is right, this does work as a standard test. Please disregard my earlier comment.